code-conventions.md

```diff
@@ -100,14 +100,39 @@ If we need to remove or alter contributed code due to a licensing issue we will
   - When omitting brackets, still indent as if the code had brackets
   - Avoid omitting brackets if it produces hard-to-read code
 * Annotations for methods and classes are placed in lines before their declarations, one per line
+* All syntaxes should have proper Skript documentation annotations. (Located in package ...skript.doc.*)
+  - When declaring Skript documentation annotations with multiple lines, seperate them into a String list aswell as placing each on a new line.
+  - When it comes to **simple short examples**, it's a better format to include indentations in the actual class for simplier reading. (See the first example in the Examples annotation).
+  - We use the format character `\t` to represent the tab character in our documentation.
```

Comment:
~~~suggestion
  - We use the format character `\t` to represent the tab character in our documentation.
  - HTML tags are supported in docs annotations but it's most likely not going to be accepted when used anywhere else than in `@Description` annotation and only simple formatting HTML tags like bold, italic etc. are accepted.
  - When using HTML characters as a non HTML characters like `<` and `>` they have to be escaped, e.g. `&lt;` and `&gt;`.
```java
@Examples({
	"command /give &lt;player&gt; &lt;item&gt; &lt;amount&gt;:",
		"\ttrigger:",
			"\t\t# Code here"
```
~~~

--- 

>   - When it comes to **simple short examples**, it's a better format to include indentations in the actual class for simplier reading. (See the first example in the Examples annotation).

Since the addition of examples above, this statement can be moved below it

Reply:
Sure, but I don't believe the example should be using the Example annotation as stated in the description, it'll most likely be denied. I think an example of it with a Description annotation would be better.

Reply:
The Example annotation I used doesn't use HTML tags but it escapes some characters that are considered HTML tags (openers and closers) to be able to use `<` and `>` in command parameters, this doesn't contradict with the point before it.
I can add an example for the point behind it using Description annotation just to be more clear